deploy/helm/dynatrace-oneagent-operator/templates/Common/application.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: dynatrace-oneagent-operator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    dynatrace: operator
+    operator: oneagent
+spec:
+  descriptor:
+    type: "Dynatrace OneAgent Operator"
+    version: "0.1.0"
+    maintainers:
+      - name: Dynatrace LLC
+        url: https://www.dynatrace.com/
+    keywords:
+      - "dynatrace"
+      - "operator"
+      - "oneagent"
+      - "k8s"
+      - "monitoring"
+      - "apm"
+    description: |
+      The Dynatrace OneAgent Operator allows users to easily deploy full-stack monitoring for [Kubernetes clusters](https://www.dynatrace.com/technologies/kubernetes-monitoring/). The Dynatrace OneAgent automatically monitors the workload running in containers down to the code and request level.
+
+      ### Installing the OneAgent
+      Once you've installed the Operator, you can create OneAgent custom resources to monitor your environment.
```

Comment:
Hmm, with the Helm chart, the OneAgent object, as well as the secret, would exist already.

Reply:
Will rephrase that - thanks for the hint